transport/src/main/java/io/netty/channel/nio/AbstractNioChannel.java

```diff
@@ -346,16 +346,6 @@ private boolean isFlushPending() {
         }
     }
 
-    @Override
-    protected void doRegister() throws Exception {
-       executor().unsafe().register(this);
-    }
-
-    @Override
-    protected void doDeregister() throws Exception {
-        executor().unsafe().deregister(this);
-    }
```

Comment:
@chrisvest where these not used anymore ?

Reply:
Their implementation were exactly the same as the super method they overrode.